ray-operator/controllers/ray/rayservice_controller_test.go

```diff
@@ -319,6 +315,53 @@ var _ = Context("Inside the default namespace", func() {
 				time.Second*3, time.Millisecond*500).Should(BeNil(), "My myRayCluster  = %v", myRayCluster.Name)
 		})
 
+		It("Autoscaler updates RayCluster and should not switch to a new RayCluster", func() {
```

Comment:
Could you add a unit test that checks that the pending RayService doesn't start a new cluster if it autoscales? I believe the current unit tests are checking only that the active RayService doesn't start a new cluster.

Reply:
Good idea! I missed the test. I have added a test "Autoscaler updates the pending RayCluster and should not switch to a new RayCluster".